packages/open-next/src/build.ts

```diff
@@ -175,33 +167,26 @@ function printOpenNextVersion(options: BuildOptions) {
   logger.info(`OpenNext v${options.openNextVersion}`);
 }
 
-function initOutputDir(srcTempDir: string, options: BuildOptions) {
+function initOutputDir(options: BuildOptions) {
   // We need to get the build relative to the cwd to find the compiled config
   // This is needed for the case where the app is a single-version monorepo and the package.json is in the root of the monorepo
   // where the build is in the app directory, but the compiled config is in the root of the monorepo.
-  const openNextConfig = readFileSync(
-    path.join(srcTempDir, "open-next.config.mjs"),
-    "utf8",
-  );
-  let openNextConfigEdge: string | null = null;
-  if (fs.existsSync(path.join(srcTempDir, "open-next.config.edge.mjs"))) {
-    openNextConfigEdge = readFileSync(
-      path.join(srcTempDir, "open-next.config.edge.mjs"),
-      "utf8",
-    );
-  }
   fs.rmSync(options.outputDir, { recursive: true, force: true });
-  const destTempDir = options.tempDir;
-  fs.mkdirSync(destTempDir, { recursive: true });
-  fs.writeFileSync(
-    path.join(destTempDir, "open-next.config.mjs"),
-    openNextConfig,
+  const { buildDir } = options;
+  fs.mkdirSync(buildDir, { recursive: true });
+
+  fs.copyFileSync(
```

Comment:
We can actually simplify by copying the whole temp folder. I'll amend the PR later today